partition-table-8M.csv

```diff
@@ -0,0 +1,8 @@
+# ESP32S3 TBeamS3Core 8M Partition Table
+# Name,   Type, SubType, Offset,  Size, Flags
+nvs,      data, nvs,    0x009000,  20K,
+otadata,  data, ota,    0x00e000,  8K,
+ota_0,    0,    ota_0,  0x010000,  2048K,
+ota_1,    0,    ota_1,  0x210000,  2048K,
+uf2,      app,  factory,0x410000,  256K,
+ffat,     data, fat,    0x450000,  3776K,
```

Comment:
The OTA Partitions need to be spaced and named like on the smaller partition table. 
```
app,      app,  ota_0,   0x010000, 0x250000,
flashApp, app,  ota_1,   0x260000, 0x0A0000,
```
The uf2 parttition can then follow at 0x300000

and i am not sure about the change to FAT. When changing from 1.2 to 2.0 development we deliberately chose littlefs to minimize flash wear. We are discussing alternatie means to access file content on this partition through protobuf and HTTP interfaces. In addition, FAT would be the completey opposite to ALL other boards we support and would create a bunch of side issues and strain support.